Compiler can't find Qt5 libraries

Can't compile currently, some Qt environment flags are not being set correctly:

    g++: error: 0/libQt5Core.so: No such file or directory
    make[3]: *** [Makefile:38: "/home/user/.cache/yay/ros-melodic-rviz/src/build/devel/lib/python3.7/site-
    packages/rviz/librviz_sip".so] Error 1
    make[2]: *** [src/python_bindings/sip/CMakeFiles/librviz_sip.dir/build.make:61: devel/lib/python3.7/site-packages/rviz/librviz_sip.so] Error 2
    make[1]: *** [CMakeFiles/Makefile2:4935: src/python_bindings/sip/CMakeFiles/librviz_sip.dir/all] Error 2
    make: *** [Makefile:130: all] Error 2

I've verified that `pyqt5-common`, `python-pyqt5`, `python-sip-pyqt5` and `qt5-base` are installed. As suggested on the AUR package page, I've tried setting QT_INSTALL_LIBS to `/usr/lib` (where the libs are), but instead I get:

    /bin/sh: /usr/lib: syntax error: operand expected (error token is "/usr/lib")

And if set QT_INSTALL_LIBS=-l/usr/lib/libQt5Core.so, I get instead:

	/bin/sh: -l/usr/lib/libQt5Core.so: division by 0 (error token is "usr/lib/libQt5Core.so")

So is there a workaround for the moment? What environment variable do I need to set?

--- a/.SRCINFO
+++ b/.SRCINFO
@@ -1,7 +1,7 @@
 pkgbase = ros-melodic-rviz
 	pkgdesc = ROS - 3D visualization tool for ROS.
-	pkgver = 1.13.3
-	pkgrel = 2
+	pkgver = 1.13.5
+	pkgrel = 1
 	url = https://wiki.ros.org/rviz
 	arch = any
 	license = BSD, Creative Commons
@@ -70,8 +70,8 @@ pkgbase = ros-melodic-rviz
 	depends = qt5-base
 	depends = sip
 	depends = python-sip
-	source = ros-melodic-rviz-1.13.3.tar.gz::https://github.com/ros-visualization/rviz/archive/1.13.3.tar.gz
-	sha256sums = c9d9f1e64246e61754ca1a44344e6e258026d660bd0d5ec1969b040fc6cbf629
+	source = ros-melodic-rviz-1.13.5.tar.gz::https://github.com/ros-visualization/rviz/archive/1.13.5.tar.gz
+	sha256sums = ec97e11644c10f7d968109d24015a752daf4a20fe463c067c5d6a369c876f3b3
 
 pkgname = ros-melodic-rviz
 

--- a/PKGBUILD
+++ b/PKGBUILD
@@ -4,9 +4,9 @@ pkgdesc="ROS - 3D visualization tool for ROS."
 url='https://wiki.ros.org/rviz'
 
 pkgname='ros-melodic-rviz'
-pkgver='1.13.3'
+pkgver='1.13.5'
 arch=('any')
-pkgrel=2
+pkgrel=1
 license=('BSD, Creative Commons')
 
 ros_makedepends=(
@@ -90,7 +90,7 @@ depends=(
 
 _dir="rviz-${pkgver}"
 source=("${pkgname}-${pkgver}.tar.gz"::"https://github.com/ros-visualization/rviz/archive/${pkgver}.tar.gz")
-sha256sums=('c9d9f1e64246e61754ca1a44344e6e258026d660bd0d5ec1969b040fc6cbf629')
+sha256sums=('ec97e11644c10f7d968109d24015a752daf4a20fe463c067c5d6a369c876f3b3')
 
 build() {
 	# Use ROS environment variables.
@@ -118,15 +118,7 @@ build() {
 		-DPYTHON_BASENAME=.cpython-37m \
 		-DSETUPTOOLS_DEB_LAYOUT=OFF
 
-	make || {
-	# Replace $$[QT_INSTALL_LIBS] with hardcoded /usr/lib/ for proper linking
-	# Fix courtesy of astier:
-	#  https://aur.archlinux.org/packages/ros-melodic-rviz/#comment-698298
-	# Relevant upstream issue:
-	#  https://github.com/ros-visualization/rviz/issues/1382
-	sed -i 's/$$\[QT_INSTALL_LIBS\]/\/usr\/lib/' ${srcdir}/build/devel/bin/sip/rviz_sip/Makefile
 	make
-	}
 }
 
 package() {

